opentelemetry-api/tests/context/test_context.py

```diff
@@ -74,3 +77,14 @@ def test_set_current(self):
 
         context.detach(token)
         self.assertEqual("yyy", context.get_value("a"))
+
+
+class TestInitContext(unittest.TestCase):
+    def test_load_runtime_context(self):
+        environ[OTEL_PYTHON_CONTEXT] = "contextvars_context"
+        ctx = context._load_runtime_context()  # pylint: disable=W0212
+        self.assertIsInstance(ctx, ContextVarsRuntimeContext)
+
+        environ.pop(OTEL_PYTHON_CONTEXT)
```

Comment:
nit I think this should be a separate test case function

Reply:
Fixed